README.md

```diff
@@ -25,8 +25,6 @@ The quick start hasn't been written yet (contributions are welcome!). The quick
 `arraybuffer-types` documentation is stored in a few places:
 
 1. Module documentation is [published on Pursuit](https://pursuit.purescript.org/packages/purescript-arraybuffer-types).
```

Comment:
Given there is only one of these left, what if we just merge this with the prior sentence? “Module documentation for arraybuffer-types ...”

Reply:
I merged the sentences per your suggestion.

I also cleaned up another spot that was referring to the `docs` directory.